src/zcl/definition/cluster.ts

```diff
@@ -5029,6 +5029,16 @@ const Cluster: {
         commands: {},
         commandsResponse: {},
     },
+    clipsalWiserFanSwitchConfigurationCluster: {
```

Comment:
Can we give it the `manuSpecific` prefix, most newer manufacturer specific clusters start with that prefix (@Koenkk we should probably at some point rename all the others, but that seems like it will require a lot of PRs against zhc too)

Also it seems the manufcaturer is schneider so probaby something like `manuSpecificSchneiderFanSwitchConfiguration`

Reply:
I'm open to whatever naming convention you all wanna go with. 

Just need it to merged so we can update the device that requires it. 👍

Reply:
@sjorge not that I don't disagree with the statements about the convention here, but I wonder if its better to do these as a big sweep and stick to the existing convention or not? If you see the cluster above this one which this is based off you'll see how its named as well as the attribute naming.

Not sure what the best approach is 👍🏼

Reply:
Not adding new ones that do follow the convention would be a good start I think.

Otherwise we're just making more extra work when we rename the others later.